migrations_server.js

```diff
@@ -302,5 +300,5 @@ Migrations._reset = function() {
 
 // unlock control
 Migrations.unlock = function() {
-  this._collection.update({ _id: 'control' }, { $set: { locked: false } });
+  this._collection.updateAsync({ _id: 'control' }, { $set: { locked: false } });
```

Comment:
missing await?
we could have async and the normal ones could stay as they are today

Reply:
We could do a feature release to be backward compatible. But since there is really no functional update (nor is there any expected soon) I don't think that is needed.